stubs/inertia/resources/js/Pages/Profile/LogoutOtherBrowserSessionsForm.vue

```diff
@@ -48,7 +48,7 @@
                     Logout Other Browser Sessions
                 </jet-button>
 
-                <jet-action-message :on="form.recentlySuccessful" class="ml-3">
+                <jet-action-message :on="formRecentlySuccessful" class="ml-3">
```

Comment:
Again, I'd be inclined to use a more meaningful name here, such as `successfullyLogoutOtherBrowserSessions`.
```suggestion
                <jet-action-message :on="successfullyLogoutOtherBrowserSessions" class="ml-3">
```